<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_left">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>&lt;-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_home">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>#</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lbl_path">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListView" name="listFiles">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_zip">
          <property name="text">
           <string>Архівувати</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_unzip">
          <property name="text">
           <string>Розархівувати</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWinBY">
    <property name="title">
     <string>WinBY</string>
    </property>
    <addaction name="separator"/>
    <addaction name="act_about"/>
    <addaction name="act_help"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вибрати файл(и)</string>
    </property>
    <addaction name="act_zip"/>
    <addaction name="act_unzip"/>
   </widget>
   <addaction name="menuWinBY"/>
   <addaction name="menu"/>
  </widget>
  <action name="act_about">
   <property name="text">
    <string>Про &quot;WinBY&quot;</string>
   </property>
  </action>
  <action name="act_help">
   <property name="text">
    <string>Допомога</string>
   </property>
  </action>
  <action name="act_zip">
   <property name="text">
    <string>Архівувати</string>
   </property>
  </action>
  <action name="act_unzip">
   <property name="text">
    <string>Розархівувати</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
